src/filter/biquad.c

```diff
@@ -11,42 +11,41 @@ void biquad_init(float filterCutFreq, biquad_axis_state_t *state, float refreshR
 
     float dbGain = 4.0;
 
-    samplingRate = (1.0f / refreshRateSeconds);
+    samplingRate = (1 / refreshRateSeconds);
 
-	omega = 2.0f * (float)M_PI_FLOAT * (float) filterCutFreq / samplingRate;
+	omega = 2 * (float)M_PI_FLOAT * (float) filterCutFreq / samplingRate;
 	sn    = (float)sinf((float)omega);
 	cs    = (float)cosf((float)omega);
-	alpha = sn * (float)sinf( (float)((float)M_LN2_FLOAT / 2.0f * (float)bandwidth * (omega / sn)) );
+	alpha = sn * (float)sinf( (float)((float)M_LN2_FLOAT / 2 * (float)bandwidth * (omega / sn)) );
 
 	(void)(beta);
 
 	switch (filterType)
 	{
 		case FILTER_TYPE_LOWPASS:
-			b0 = (1.0f - cs) * 0.5f;
-			b1 = 1.0f - cs;
-			b2 = (1.0f - cs) * 0.5f;
-			a0 = 1.0f + alpha;
-			a1 = -2.0f * cs;
-			a2 = 1.0f - alpha;
+			b0 = (1 - cs) /2;
+			b1 = 1 - cs;
+			b2 = (1 - cs) /2;
+			a0 = 1 + alpha;
+			a1 = -2 * cs;
+			a2 = 1 - alpha;
 			break;
 		case FILTER_TYPE_NOTCH:
-			b0 = 1.0f;
-			b1 = -2.0f * cs;
-			b2 = 1.0f;
-			a0 = 1.0f + alpha;
-			a1 = -2.0f * cs;
-			a2 = 1.0f - alpha;
+			b0 = 1;
+			b1 = -2 * cs;
+			b2 = 1;
+			a0 = 1 + alpha;
+			a1 = -2 * cs;
+			a2 = 1 - alpha;
 			break;
 	}
 	//don't let these states be used until they're updated
 	__disable_irq();
-		const float a0r = 1.0f / a0;
```

Comment:
please revert this, it runs faster this way, if needed look at the math. Its the same but faster.